article.md

```diff
@@ -0,0 +1,266 @@
+# How to build Cloud Foundry Eirini Extensions with Eirinix
+
```

Comment:
maybe you could add a disclaimer mentioning that previous knowledge about Eirini is required and where to find out